help/restrict-account-creation.md

```diff
@@ -51,17 +50,16 @@ Regardless of whether invitations are required, you can:
 
 {!owner-only.md!}
 
-You can restrict the ability to invite new users to join your Zulip organization
-to specific [roles](/help/roles-and-permissions). To protect your organization,
-while permission to send out individual email invitations is configurable, creating
-*reusable* invitation links is always limited to administrators.
+To protect your organization, permissions to send out individual email
+invitations and creating reusable invite links are configurable.
```

Comment:
Again, I think we can just delete all the content before the header here, even the `{!owner-only.md!)` since that's at the top of this page.